README.md

```diff
@@ -128,6 +128,13 @@ To install this gem onto your local machine, run `bundle exec rake install`.
 
 To release a new version, follow the instructions in [PUBLISHING.md](PUBLISHING.md).
 
+### Nix support
+
+In order to get Nix working you need to have [direnv](https://github.com/direnv/direnv) installed. You can then run from the top level folder:
+
+```
+cd nix && bundix -m --ruby=ruby_2_5 && cd .. && direnv reload
```

Comment:
only one more thing: it's customary to have a script named `update.sh` that will do this dance for you. Hard to remember the flags otherwise and this documentation will drift out of date. I'd suggest `./nix/update-gemset.sh` in this case. That'd look something like this:

```sh
#!/usr/bin/env nix-shell
#!nix-shell -i bash -p zlib sqlite libiconv bundix
# TODO: `cd` to this script's directory somehow. I forget the one-liner.
bundix -m --ruby=ruby_2_5
```

then you can get rid of the build deps in `shell.nix` as well. If you use `nix-shell` as the interpreter it can load dependencies in a subsequent `nix-shell`-specific shebang.

Reply:
Amazing, great suggestion!